zfit/core/constraint.py

```diff
@@ -85,14 +78,14 @@ def _get_dependents(self):
             self._simple_func_dependents = dependents
         return dependents
 
-    def _eval(self):
+    def _value(self):
         return self._simple_func()
 
 
-class ProbConstraint(BaseConstraint):
+class DistributionConstraint(BaseConstraint):
 
-    def __init__(self, params: Dict[str, ZfitParameter], distribution: Distribution,
-                 name: str = "ProbConstraint", dtype=ztypes.float,
+    def __init__(self, params: Dict[str, ZfitParameter], distribution: tfd.Distribution,
+                 dist_params, dist_kwargs=None, name: str = "DistributionConstraint", dtype=ztypes.float,
                  **kwargs):
```

Comment:
Default missing: `if dist_kwargs is None` use `{}` (otherwise fails in the distribution property)

Reply:
You would rather have `dist_kwargs={}` in the function arguments or inside the function
```
if dist_kwargs is None:
    dist_kwargs = {}
```

Reply:
Inside is fine, as there is always the issue of the mutable default arg.
So let's keep None and convert inside, agree?